app/views/miq_ae_customization/_dialog_field_form_dynamic_options.html.haml

```diff
@@ -28,4 +28,91 @@
     = check_box_tag('field_auto_refresh', '1',
                     @edit[:field_auto_refresh],
                     'data-miq_observe_checkbox' => {:url => url}.to_json)
+.form-group
```

Comment:
This one stands out to me, but a lot of these `.form-group`s are duplicated now, right? Should we maybe be looking at separating them into their own partials in the future so that we don't accidentally forget to change both places?